docs/SuperSIDonRaspi.md

```diff
@@ -1,20 +1,38 @@
 # SuperSID on Raspberry Pi with Rapberry OS
 
+This has been tested on
+
+Raspberry Pi 3 Model B
+Raspberry Pi 3 Model B+
+Raspberry Pi 4 Model B
+Raspberry Pi Zero W
+Raspberry Pi Zero 2 W
+
 ## Preparation
 
-[Set up your Raspberry Pi](https://www.raspberrypi.com/documentation/computers/getting-started.html#setting-up-your-raspberry-pi) with the image **Raspberry OS (32-bit)**. At the date of the installation this corresponds to *buster*.
-Boot on the new micro-SD card, follow normal process for any fresh system install. Connect to the internet.
+[Set up your Raspberry Pi]
+(https://www.raspberrypi.com/documentation/computers/getting-started.html#setting-up-your-raspberry-pi)
+with the image **Raspberry OS (32-bit)**. At the date of the installation this
+corresponds to *buster*. Boot on the new micro-SD card,
```

Comment:
Current is now "bullseye". Maybe mention the date in this sentence? like 
"Current version as of 2022-01-03 is bullseye"

Reply:
I installed and test on 'buster' end of October 2021. Thus I'm describing buster. It doesn't feel right to describe a version I'm not testing with.

Reply:
Okay. I ran through the steps in SuperSIDonRaspi.md a few days ago 12/29/2021 and it installed bullseye, which seemed to work just fine. I found that a few of the steps in this document were no longer needed.
For a new user just installing on a clean Raspberry PI, I'd expect them to be installing whatever the latest is, so our instructions should reflect that as best we can.

Reply:
Ok, I'll add a clarification to SuperSIDonRaspi.md. See next comment.